src/styles/_mixins.scss

```diff
@@ -0,0 +1,13 @@
+@import "./../components/variables";
```

Comment:
We don't need to use underscores in `.scss` file names, because we don't have an auto import of them.